docs/clients/brushless_drive.rst

```diff
@@ -105,121 +105,121 @@ Message Table
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 12     | obs_angle                     | get            | float     | rad      | Observed motor angle                                                                                                                                                                                                                                                                                                                                                                                        |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 13     | obs_velocity                  | get            | float     | rad/s    | Observed motor velocity                                                                                                                                                                                                                                                                                                                                                                                     |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 14     | motor_pole_pairs              | get, set, save | uint16    | PP       | Number of motor pole pairs (magnets/2)                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
do we need to specify the (magnets/2) part? if we do, can we change it to (number of rotor magnets / 2)? @iq-matt

Reply:
Either ok with me.  Slight preference to removing.